### ISSUE TITLE
EOC multiplier and divider

### ISSUE THREAD

**Issue by patheros** [patheros/PathSetXOmriCohen#21]

**Comment by patheros** [comment]
@omricohenmusic Questions:
1. Is this in the right click menu or on the panel?
2. What options would you like it to have?

Just a note, this seems not needed since there are a lot of existing clock dividers and multipliers, but I understand how it might be useful.

**Comment by omricohenmusic** [comment]
Yeah, in the right click menu. We did this for Prince\Prism of perception, and it works really well for syncing loops with other modules. I guess that 2 to 16 will be enough.
